service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/CreateAiNotebookInstanceStep.java

```diff
@@ -45,6 +45,10 @@
  * <p>Undo deletes the created notebook instance.
  */
 public class CreateAiNotebookInstanceStep implements Step {
+
+  /** Default post-startup-script when starting a notebook instance. */
+  protected static final String DEFAULT_POST_STARTUP_SCRIPT =
+      "https://raw.githubusercontent.com/DataBiosphere/terra-workspace-manager/main/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh";
```

Comment:
Could/should this be in a configuration file? Do we think we'll ever need to customize that script per-environment?

Reply:
It does not have environment specific content *yet*. I'd say we keep it simple for now and move it when we need to?